Jenkinsfile

```diff
@@ -71,55 +70,77 @@ def pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos')
 }
 
-String el7_component_repos() {
+String el7_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-el7')
 }
 
-String leap15_component_repos() {
+String leap15_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-leap15')
 }
 
-def daos_repo() {
+String unit_repos() {
+    Map stage_info = parseStageInfo()
+    return unit_repos(stage_info['target'])
+}
+
+String unit_repos(String distro) {
+    if (distro == 'centos7') {
+        return el7_pr_repos() + pr_repos()
```

Comment:
Isn't this going to result in the value of both of the commit pragmas smushed together with no space between them?

FWIW, I have been considering that we use space separated lists frequently where we really do want lists and should use lists and only expand them into space separated lists when that's what they need to be.  Then this kind of natural concatenation would be more natural.

Reply:
Will fix in the next push.